sim/src/main/makefrag/firesim/Makefrag

```diff
@@ -96,7 +97,7 @@ DRIVER_H = \
 	$(TESTCHIPIP_CSRC_DIR)/testchip_tsi.h
 
 DRIVER_CC = \
-	    $(addprefix $(driver_dir)/firesim/, $(addsuffix .cc, firesim_top systematic_scheduler firesim_$(PLATFORM))) \
+		$(addprefix $(driver_dir)/firesim/, $(addsuffix .cc, firesim_top systematic_scheduler firesim_$(PLATFORM))) \
```

Comment:
I would just revert this file completely to make the diff nice/obsolete.

Reply:
Looking at this line more carefully, they aren't the same. I'll revert everything else though.